include/triton/core/tritonserver.h

```diff
@@ -1071,6 +1071,20 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceRequestAddInput(
     const TRITONSERVER_DataType datatype, const int64_t* shape,
     uint64_t dim_count);
 
+/// Add a raw input to a request. The model input name, data type and shape of
```

Comment:
Redundant to have input twice, 

"The model input name, data type and shape will be ..."

Reply:
Can you elaborate?

Reply:
The statement starts with model input name ... then mentions shape of input again. 

My suggestion was to remove the second occurrence of input

Reply:
Something like: "The name, data type and shape of the model input will be deduced from model configuration."

Reply:
Maybe "the name recognized by the model" then, "model input name" is used to distinguish from the use of  "name" below